server/hap2/hatohol/zabbixapi.py

```diff
@@ -311,7 +311,7 @@ def get_event_end_id(self, is_first=False):
         if not self.result:
             return
 
-        return int(res_dict[0]["eventid"])
+        return res_dict[0]["eventid"]
```

Comment:
I think this line assumes always Zabbix API returning correct json result.
We should add assertion or error in this line? Or, just ignored it for now?